common/config/src/argv.rs

```diff
@@ -281,6 +287,10 @@ impl ArgumentOptions {
             raw.http.retry_dir = self.retry_dir.map(PathBuf::from);
         }
 
+        if self.retry_disk_limit.is_some() {
+            raw.http.retry_disk_limit = self.retry_disk_limit;
```

Comment:
Does setting it to `0` mean storing retry files won't be allowed? 
If that's the case, it's a nice little feature that would be useful to document.

Reply:
Correct. It acts as a disable flag.

Reply:
> Should we default to a super super large value, like 2 or 4GiB.
> I think having unlimited storage usage by default is unexpected and during long outages can lead to infra issues on the user side.

We discussed this at standup today and agree that filling the disk space up is problematic but that enforcing a default might lead to data loss if we pick incorrectly. Customers who are sensitive to their disk usage can monitor the disk used as well as set limits for their configurations.